scipy/stats/tests/test_stats.py

```diff
@@ -6009,17 +6060,17 @@ def test_describe_numbers(self, xp):
         assert n == nc
         xp_assert_equal(mm[0], mmc[0])
         xp_assert_equal(mm[1], mmc[1])
-        xp_assert_equal(m, mc)
-        xp_assert_equal(v, vc)
```

Comment:
I was seeing

```
________________ TestDescribe.test_describe_numbers[jax.numpy] _________________
scipy/stats/tests/test_stats.py:6063: in test_describe_numbers
    xp_assert_equal(m, mc)
        kurt       = Array([-1.83333333, -1.83333333, -1.83333333, -1.83333333], dtype=float64)
        kurtc      = Array([-1.83333333, -1.83333333, -1.83333333, -1.83333333], dtype=float64)
        m          = Array([1.4, 1.4, 1.4, 1.4], dtype=float64)
        mc         = Array([1.4, 1.4, 1.4, 1.4], dtype=float64)
        mm         = (Array([1., 1., 1., 1.], dtype=float64), Array([2., 2., 2., 2.], dtype=float64))
        mmc        = (Array([1., 1., 1., 1.], dtype=float64), Array([2., 2., 2., 2.], dtype=float64))
        n          = 5
        nc         = 5
        self       = <scipy.stats.tests.test_stats.TestDescribe object at 0x7f2fc37d8a90>
        sk         = Array([0.40824829, 0.40824829, 0.40824829, 0.40824829], dtype=float64)
        skc        = Array([0.40824829, 0.40824829, 0.40824829, 0.40824829], dtype=float64)
        v          = Array([0.3, 0.3, 0.3, 0.3], dtype=float64)
        vc         = Array([0.3, 0.3, 0.3, 0.3], dtype=float64)
        x          = Array([[1., 1., 1., 1.],
       [1., 1., 1., 1.],
       [1., 1., 1., 1.],
       [2., 2., 2., 2.],
       [2., 2., 2., 2.]], dtype=float64)
        xp         = <module 'jax.numpy' from '/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/jax/numpy/__init__.py'>
        xp_test    = <module 'jax.experimental.array_api' from '/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/jax/experimental/array_api/__init__.py'>
/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/contextlib.py:81: in inner
    return func(*args, **kwds)
E   AssertionError: 
E   Arrays are not equal
E   
E   Mismatched elements: 4 / 4 (100%)
E   Max absolute difference: 2.22044605e-16
E   Max relative difference: 1.58603289e-16
E    x: array([1.4, 1.4, 1.4, 1.4])
E    y: array([1.4, 1.4, 1.4, 1.4])
```

Reply:
I'd say that `assert_close` is better anyway than exact equality comparison for floating-point arrays. However, this shouldn't use the default tolerance. The difference is exactly 1 ULP here it looks like. So how about on the next line, you use:
```python
xp_assert_close(m, mc, rtol=4 * xp.finfo(m.dtype).eps)
```
and same for the line below it? With the added margin, that should work. Or if there's no `float32` version to worry about, `rtol=1e-15` will do.